plugins/common/view-action-interface.cpp

```diff
@@ -0,0 +1,308 @@
+#include "wayfire/output.hpp"
```

Comment:
I have a feeling that this file belongs exclusively to window-rules, doesn't it? So it shouldn't be in common.

Reply:
I put it in common because other plugins that use the rule engine may use it. Also, window-rules was a single plugin and I did not want to add it there. I will move window-rules to a folder as you suggested, but we should still weigh potential reuse of the view access interface versus the grouping of all files. I am fine with either decision, but I did have a good reason to put that in common.

Reply:
Alright, but view-access-interface is in core, right? It is reusable by all plugins already.

Reply:
Done